CHANGES.rst

```diff
@@ -6,6 +6,10 @@ New Features
 
 API Changes
 ^^^^^^^^^^^
+Previously, the ``setup`` and ``teardown`` methods were run only once
+even when the benchmark had ``repeat > 1`` specified. This is now
+changed so that they are run on every repeat, and additionally before
+and after profiling runs.
```

Comment:
But the behavior with profiling runs is unchanged, correct?  If so, maybe it's less confusing to not mention it.